src/encoding/schema/map.ts

```diff
@@ -424,3 +449,129 @@ export class StringMapSchema extends Schema {
     return map;
   }
 }
+
+function removeRawStringsFromMsgpackValues(
```

Comment:
not sure I understand "remove" part in its name - isn't it converts raw strings instances to bytes?

Reply:
Right, this is not the best name. Updated in 32c59407